src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -901,12 +897,8 @@ private static String replaceUnpredictableValuesWithPlaceholders(String content)
                 // today's date
                 .replace(TimeHelper.formatDate(now).replace("/", "&#x2f;"), "${today}")
                 .replace(TimeHelper.formatDate(now), "${today}")
-                // now (used in comments last edited date) e.g. [Thu, 07 May 2015, 07:52:13 UTC]
-                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC", "\\${comment\\.date}")
-                // now (date, time)
-                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2} [AP]M", "\\${datetime\\.now}")
-                // dynamic feedback submission numbers
-                .replaceAll("(?s)<span class=\"submissionsNumber\".*?</span>", "<span class=\"submissionsNumber\" id=\"submissionsNumber\">\\${submissions\\.number}</span>")
+                // now (date, time) e.g. [Thu, 07 May 2015, 07:52 PM] or [Thu, 07 May 2015, 07:52:13 UTC]
+                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}( [AP]M|:[0-9]{2} UTC)", "\\${datetime\\.now}")
```

Comment:
1. Just because we standardize the date/time format, it does _not_ necessarily translate to the same placeholder used (see how our regkey, question ID, ... have the same regex?). It will be breaking abstraction, something like "it just happened that the same date/time format is used in both places". Then why do I standardize the placeholder? Like I mentioned in the issue thread, `${comment.date}` makes little to no sense and it really refers to date/time now formatted for comments.
2. In both cases of truncated accounts and different date/time formats, the placeholders are _not_ the only place where the different formats are tested. In the former case we have a test case with hard-coded long account to test whether the truncation occurs. In the latter case, we have _hundreds_ of instances of date/time format of comments. If the placeholder fails its job, the hard-coded instances will cover it.

Then why is standardization of date/time format involved here? Once again, standardization is never my primary objective; optimization is, and it is achieved by reducing the number of `.replace(...)`. `${comment.date}` happens to be one that can be removed. I can either do this:
```
.replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC", "\\${datetime\\.now}")
.replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2} [AP]M", "\\${datetime\\.now}")
```
which has no performance boost, or this (currently):
```
.replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}( [AP]M|:[0-9]{2} UTC)", "\\${datetime\\.now}")
```
which might have little to no performance boost I'm not sure, or this (what I'm aiming for):
```
.replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2} [AP]M( UTC)?", "\\${datetime\\.now}")
```
which has obvious performance boost.